data/loans.json

```diff
@@ -82,5 +82,14 @@
       "rank_2": 1,
       "requests": 2
     }
+  },
```

Comment:
Only one loan has been added for the second organisation?

Reply:
data provided by PO